superset-frontend/src/setup/setupColors.ts

```diff
@@ -16,36 +16,39 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import airbnb from '@superset-ui/core/esm/color/colorSchemes/categorical/airbnb';
-import categoricalD3 from '@superset-ui/core/esm/color/colorSchemes/categorical/d3';
-import echarts from '@superset-ui/core/esm/color/colorSchemes/categorical/echarts';
-import google from '@superset-ui/core/esm/color/colorSchemes/categorical/google';
-import lyft from '@superset-ui/core/esm/color/colorSchemes/categorical/lyft';
-import preset from '@superset-ui/core/esm/color/colorSchemes/categorical/preset';
-import sequentialCommon from '@superset-ui/core/esm/color/colorSchemes/sequential/common';
-import sequentialD3 from '@superset-ui/core/esm/color/colorSchemes/sequential/d3';
+import airbnb from '@superset-ui/core/lib/color/colorSchemes/categorical/airbnb';
+import categoricalD3 from '@superset-ui/core/lib/color/colorSchemes/categorical/d3';
+import echarts from '@superset-ui/core/lib/color/colorSchemes/categorical/echarts';
+import google from '@superset-ui/core/lib/color/colorSchemes/categorical/google';
+import lyft from '@superset-ui/core/lib/color/colorSchemes/categorical/lyft';
+import preset from '@superset-ui/core/lib/color/colorSchemes/categorical/preset';
+import sequentialCommon from '@superset-ui/core/lib/color/colorSchemes/sequential/common';
+import sequentialD3 from '@superset-ui/core/lib/color/colorSchemes/sequential/d3';
 import {
+  CategoricalScheme,
   getCategoricalSchemeRegistry,
   getSequentialSchemeRegistry,
+  SequentialScheme,
 } from '@superset-ui/core';
-import superset from '@superset-ui/core/esm/color/colorSchemes/categorical/superset';
+import superset from '@superset-ui/core/lib/color/colorSchemes/categorical/superset';
+import ColorScheme from '@superset-ui/core/lib/color/ColorScheme';
 
 export default function setupColors(
-  extraCategoricalColorSchemas,
-  extraSequentialColorSchemes,
+  extraCategoricalColorSchemas: ColorScheme[] = [],
+  extraSequentialColorSchemes: ColorScheme[] = [],
```

Comment:
Should this be
```suggestion
  extraCategoricalColorSchemas: CategoricalColorScheme[] = [],
  extraSequentialColorSchemes: SequentialColorScheme[] = [],
```

Reply:
These interfaces are not imported from superset-ui
![image](https://user-images.githubusercontent.com/56388545/106440609-b5e25580-6481-11eb-95f8-56e10395aa9b.png)